python/src/Wishart_doc.i.in

```diff
@@ -5,9 +5,13 @@ Parameters
 ----------
 V : 2-d sequence of float
     Scale matrix, positive definite of size :math:`p`.
+
+    Default value is :math:`[[1.0]]`.
```

Comment:
The doubled squared brackets do not refer to any mathematical definition, but to the syntax we use to create a `Sample` from a Python `list`. This is why I suggest to use:
```suggestion
    Default value is `[[1.0]]`.
```

Reply:
ok